isis/tests/FunctionalTestsMsi2isis.cpp

```diff
@@ -0,0 +1,469 @@
+#include <QTemporaryDir>
+
+#include "msi2isis.h"
+#include "Fixtures.h"
+#include "Pvl.h"
+#include "PvlGroup.h"
+#include "TestUtilities.h"
+#include "Histogram.h"
+
+#include "gtest/gtest.h"
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/msi2isis.xml").expanded();
+
+TEST(Msi2isis, Msi2isisTestGblIngestDefault) {
+  Pvl appLog;
+  QTemporaryDir prefix;
```

Comment:
Use the `TempTestingFiles` fixture instead of making your own temp directory. This makes it easier for us to modify how we handle test outputs and temp files in the future.

Reply:
Done. This changes the test name patterns a bit:
```
(Mar282022) zion2[1589]: ctest -R Msi2isis
Test project /Users/kbecker/ISIS/Isis4Dev/Eros/MSI/Mar282022/ISIS3/build
      Start 1710: TempTestingFiles.Msi2isisTestGblIngestDefault
 1/13 Test #1710: TempTestingFiles.Msi2isisTestGblIngestDefault ..........   Passed    0.56 sec
      Start 1711: TempTestingFiles.Msi2isisTestGblIngestNoTrim
 2/13 Test #1711: TempTestingFiles.Msi2isisTestGblIngestNoTrim ...........   Passed    0.34 sec
      Start 1712: TempTestingFiles.Msi2isisTestCubicConvolution
 3/13 Test #1712: TempTestingFiles.Msi2isisTestCubicConvolution ..........   Passed    0.50 sec
      Start 1713: TempTestingFiles.Msi2isisTestNearestNeighbor
 4/13 Test #1713: TempTestingFiles.Msi2isisTestNearestNeighbor ...........   Passed    0.47 sec
      Start 1714: TempTestingFiles.Msi2isisTestBilinear
 5/13 Test #1714: TempTestingFiles.Msi2isisTestBilinear ..................   Passed    0.49 sec
      Start 1715: TempTestingFiles.Msi2isisTestAMissingLabelFile
 6/13 Test #1715: TempTestingFiles.Msi2isisTestAMissingLabelFile .........   Passed    0.18 sec
      Start 1716: TempTestingFiles.Msi2isisTestBInvalidInstrumentId
 7/13 Test #1716: TempTestingFiles.Msi2isisTestBInvalidInstrumentId ......   Passed    0.18 sec
      Start 1717: TempTestingFiles.Msi2isisTestCInvalidSize
 8/13 Test #1717: TempTestingFiles.Msi2isisTestCInvalidSize ..............   Passed    0.17 sec
      Start 1718: TempTestingFiles.Msi2isisTestDInvalidSize2
 9/13 Test #1718: TempTestingFiles.Msi2isisTestDInvalidSize2 .............   Passed    0.18 sec
      Start 1719: TempTestingFiles.Msi2isisTestEInvalidSampleDirection
10/13 Test #1719: TempTestingFiles.Msi2isisTestEInvalidSampleDirection ...   Passed    0.17 sec
      Start 1720: TempTestingFiles.Msi2isisTestFInvalidLineDirection
11/13 Test #1720: TempTestingFiles.Msi2isisTestFInvalidLineDirection .....   Passed    0.17 sec
      Start 1721: TempTestingFiles.Msi2isisTestGInvalidProjectedImage
12/13 Test #1721: TempTestingFiles.Msi2isisTestGInvalidProjectedImage ....   Passed    0.17 sec
      Start 1722: TempTestingFiles.Msi2isisTestHLabelTranslateError
13/13 Test #1722: TempTestingFiles.Msi2isisTestHLabelTranslateError ......   Passed    0.36 sec

100% tests passed, 0 tests failed out of 13

Total Test time (real) =   4.10 sec
```